src/tcgwars/logic/impl/gen3/UnseenForces.groovy

```diff
@@ -2602,13 +2602,13 @@ public enum UnseenForces implements LogicCardInfo {
               for (enCard in self.cards.filterByType(ENERGY)){
                 def enTypes = enCard.getEnergyTypes()
                 if (enTypes.size() == 1){
-                  potentialEnergy.add([enTypes.first(), enCard, "${enCard}"])
+                  potentialEnergy.add([enTypes.first(), enCard, "${enCard}"],0)
                 } else {
                   def i = 1
                   def total = enTypes.size()
                   //TODO: Cover multiple of the same energy, add some identifier for e.g. 2 Holon's Castform
                   for (enTypeSet in enTypes) {
-                    potentialEnergy.add([enTypeSet, enCard, "${enCard} - Energy #${i}/${total}"])
+                    potentialEnergy.add([enTypeSet, enCard, "${enCard} - Energy #${i}/${total}"],total-i)
```

Comment:
```suggestion
                    potentialEnergy.add([enTypeSet, enCard, "${enCard} - Energy #${i}/${total}",total-i-1])
```

Initially just moved the square bracket, but prev value would do 1-2 instead of 0-1 (for a double energy)